src/instrumenter/instrument.ts

```diff
@@ -978,15 +1017,35 @@ function insertVarsStruct(
 }
 
 function getCheckStateInvsFuncs(contract: ContractDefinition): FunctionDefinition {
-    return single(contract.vFunctions.filter((fn) => fn.name === CHECK_STATE_INVS_FUN));
```

Comment:
This function is a bit fragile. Instead of relying on reg-exes to find the the check invariants functions, and duplicating the knowledge of the exact shape of those functions, lets remove this function all-toghether, and instrumentedInvariantData as an argument to `FunctionInstrumenter.instrument`. This is the only place where this function is used.